src/frontends/pytorch/src/transforms/string_equality_replacer.cpp

```diff
@@ -0,0 +1,86 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "string_equality_replacer.hpp"
+
+#include "openvino/core/rt_info.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/convert_like.hpp"
+#include "openvino/op/equal.hpp"
+#include "openvino/op/not_equal.hpp"
+#include "openvino/pass/pattern/matcher.hpp"
+#include "openvino/pass/pattern/op/or.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "pt_framework_node.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace pass {
+
+using namespace ov::pass;
+using namespace ov::op;
+
+std::string get_str_val(std::shared_ptr<PtFrameworkNode> fw_node) {
+    if (!fw_node) {
+        return nullptr;
```

Comment:
Does this statement even compile? I think it should either return an empty string or throw an exception in case the required conditions are not met by the node passed in.

Reply:
Changed to `FRONT_END_OP_CONVERSION_CHECK`.